frontend/public/components/dashboard/project-dashboard/utilization-card.tsx

```diff
@@ -4,225 +4,168 @@ import DashboardCard from '@console/shared/src/components/dashboard/dashboard-ca
+      <UtilizationBody timestamps={timestamps}>
+        <PrometheusUtilizationItem
           title="CPU"
-          current={current}
-          consumers={[
-            {
-              query: getTopConsumerQueries(projectName)[ProjectQueries.PODS_BY_CPU],
-              model: PodModel,
-              metric: 'pod',
-            },
-          ]}
-          humanize={humanizeCpuCores}
-          namespace={projectName}
-          position={PopoverPosition.top}
+          humanizeValue={humanizeCpuCores}
+          utilizationQuery={queries[ProjectQueries.CPU_USAGE]}
+          TopConsumerPopover={cpuPopover}
+          duration={duration}
+          setTimestamps={setTimestamps}
         />
-      ),
-      [projectName],
-    );
-
-    const memPopover = React.useCallback(
-      ({ current }) => (
-        <ConsumerPopover
+        <PrometheusUtilizationItem
           title="Memory"
-          current={current}
-          consumers={[
-            {
-              query: getTopConsumerQueries(projectName)[ProjectQueries.PODS_BY_MEMORY],
-              model: PodModel,
-              metric: 'pod',
-            },
-          ]}
-          humanize={humanizeBinaryBytes}
-          namespace={projectName}
-          position={PopoverPosition.top}
+          humanizeValue={humanizeBinaryBytes}
+          utilizationQuery={queries[ProjectQueries.MEMORY_USAGE]}
+          byteDataType={ByteDataTypes.BinaryBytes}
+          TopConsumerPopover={memPopover}
+          duration={duration}
         />
-      ),
-      [projectName],
-    );
-
-    const filesystemPopover = React.useCallback(
-      ({ current }) => (
-        <ConsumerPopover
+        <PrometheusUtilizationItem
           title="Filesystem"
-          current={current}
-          consumers={[
-            {
-              query: getTopConsumerQueries(projectName)[ProjectQueries.PODS_BY_FILESYSTEM],
-              model: PodModel,
-              metric: 'pod',
-            },
-          ]}
-          humanize={humanizeBinaryBytes}
-          namespace={projectName}
-          position={PopoverPosition.top}
+          humanizeValue={humanizeBinaryBytes}
+          utilizationQuery={queries[ProjectQueries.FILESYSTEM_USAGE]}
+          byteDataType={ByteDataTypes.BinaryBytes}
+          TopConsumerPopover={filesystemPopover}
+          duration={duration}
         />
-      ),
-      [projectName],
-    );
-
-    const networkPopover = React.useCallback(
-      ({ current }) => (
-        <ConsumerPopover
-          title="Network"
-          current={current}
-          consumers={[
-            {
-              query: getTopConsumerQueries(projectName)[ProjectQueries.PODS_BY_NETWORK],
-              model: PodModel,
-              metric: 'pod',
-            },
-          ]}
-          humanize={humanizeDecimalBytesPerSec}
-          namespace={projectName}
-          position={PopoverPosition.top}
+        <PrometheusUtilizationItem
+          title="Network Transfer in/out"
```

Comment:
`IN/OUT` instead?

Reply:
we should have just a `Network transfer` here, so I changed it accordingly.